soco/data_structures.py

```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
 
         return elt
 
+    def get_uri(self, resource_nr=0):
+        """The uri to use for playing this item.
```

Comment:
According to pep 257, the first single line description should preferably be kept on the form of describing the methods effect e.g. """Return the uri .."""